weaviate/graphql/neardepthbuilder.go

```diff
@@ -74,3 +76,28 @@ func (b *NearDepthArgumentBuilder) build() string {
 	builder.withTargets(b.targets)
 	return builder.build()
 }
+
+func (b *NearDepthArgumentBuilder) togrpc() *pb.NearDepthSearch {
+	builder := &nearMediaArgumentBuilder{
+		data:       b.depth,
+		dataReader: b.depthReader,
+	}
+	nearDepth := &pb.NearDepthSearch{
+		Depth: builder.getContent(),
+	}
+	if b.hasCertainty {
+		certainty := float64(b.certainty)
+		nearDepth.Certainty = &certainty
+	}
+	if b.hasDistance {
+		distance := float64(b.distance)
+		nearDepth.Distance = &distance
+	}
+	if b.targets != nil {
+		nearDepth.Targets = b.targets.togrpc()
+	}
+	if len(b.targetVectors) > 0 && b.targets == nil {
+		nearDepth.Targets = &pb.Targets{TargetVectors: b.targetVectors}
+	}
```

Comment:
```suggestion
	if b.targets != nil {
		nearDepth.Targets = b.targets.togrpc()
	} else if len(b.targetVectors) > 0 {
		nearDepth.Targets = &pb.Targets{TargetVectors: b.targetVectors}
	}
```

nit: simpler condition